modules/git/repo_tag.go

```diff
@@ -160,24 +161,18 @@ func (repo *Repository) GetTag(name string) (*Tag, error) {
 }
 
 // GetTagInfos returns all tag infos of the repository.
-func (repo *Repository) GetTagInfos(page, pageSize int) ([]*Tag, error) {
+func (repo *Repository) GetTagInfos(page, pageSize int) ([]*Tag, int, error) {
 	// TODO this a slow implementation, makes one git command per tag
```

Comment:
Probably need to fix this if we're going to offer to page this.

Reply:
what do you mean by fix? the git implementation to get tag info to truly paginate?

Reply:
leave it for this PR.

Reply:
heh I'm aware of that and the current code too - see TODO :sweat_smile: